components/automate-chef-io/data/docs/cli_chef_automate/commands/chef-automate_admin-token.yaml

```diff
@@ -1,31 +0,0 @@
-name: chef-automate admin-token
```

Comment:
Heads up: I think this file will get re-committed to the repository after merge because we have automation that autogenerates it.

Reply:
the `admin-token` command has already been removed from the CLI. should I remove it from anywhere else?

Reply:
I'm a moron and checked the wrong branch. I thought this was against master 🤦‍♂

Reply:
No worries! Don't want to miss anything 👀